### ISSUE TITLE
Separate input and select styles using Aphrodite

### ISSUE THREAD

**Issue by jkup** [brave/browser-laptop#5972]
The goal for this issue would be to separate input and select styles (which are currently coupled together).  [Per our style guidelines](https://github.com/brave/browser-laptop/blob/master/docs/style.md), solving this issue would involve rolling the input and select elements into an Aphrodite component.

- [x] Text entry field (editable)
- [x] Text entry field (focus outline)
- [x] Text entry field (locked/fixed value)
- [x] drop-down menu style
- [x] preferred drop down arrow style
- [x] flatten out the orange button
- [x] update about:styles

Here are some spec images:

## Text entry field (editable)
![image](https://cloud.githubusercontent.com/assets/13509546/20814686/99b689e8-b7cf-11e6-911a-be2191ebacaa.png)

![image](https://cloud.githubusercontent.com/assets/13509546/20814652/799babde-b7cf-11e6-8327-90a06a2fd6c5.png)

## Text entry field (focus outline)
![image](https://cloud.githubusercontent.com/assets/13509546/20814866/58afe2fe-b7d0-11e6-85b4-98a7af7baa90.png)
 (use a 2 px radius, and "@statsBlue: #0796fa;")

## Text entry field (locked/fixed value)
![image](https://cloud.githubusercontent.com/assets/13509546/20814735/cdaf0d74-b7cf-11e6-9281-f30aafc5f2cb.png)

![image](https://cloud.githubusercontent.com/assets/13509546/20814764/e29172ae-b7cf-11e6-9990-5d683978b917.png)
(use variable color for fill: @veryLightGray: rgb(250, 250, 250); )

## drop-down menu style

![image](https://cloud.githubusercontent.com/assets/13509546/20816683/6e2fce0c-b7d8-11e6-986b-ee41e271c3c4.png)

![image](https://cloud.githubusercontent.com/assets/13509546/20816703/819e374e-b7d8-11e6-8772-436a59570cda.png)

## preferred drop down arrow style
![image](https://cloud.githubusercontent.com/assets/13509546/20817665/e25edd36-b7dd-11e6-8300-86e7b6b91afe.png)

We can switch to a different FA triangle
![image](https://cloud.githubusercontent.com/assets/13509546/20817708/1aef374a-b7de-11e6-8475-7430e940f484.png)

## and this will flatten out the orange button
![image](https://cloud.githubusercontent.com/assets/13509546/20818687/1752bfe8-b7e4-11e6-8e02-ec6742fdc7c1.png)

![image](https://cloud.githubusercontent.com/assets/13509546/20818718/4be41d6a-b7e4-11e6-90d2-35c20fcec4cd.png)

**Comment by cezaraugusto** [comment]
sorry don't get it, what's the issue here?

**Comment by bsclifton** [comment]
Updated issue to capture the requirement for the Aphrodite library.  A [PR was submitted to solve the style issues](https://github.com/brave/browser-laptop/pull/6084), but we've been fighting style regressions.  Styles are very hard to test / review because of the global impact to the app.

[Our new reviewer guidelines](https://github.com/brave/browser-laptop/wiki/Reviewer-guideline) outline the need to use Aphrodite, which is why I reworked this issue after closing the mentioned PR
